File: 1-Iniciante/Media-3/1040.c
# include <stdio.h>

int main() {

    double a, b, c, d, m;
    scanf ("%lf %lf %lf %lf", &a, &b, &c, &d);
    m = (a * 2 + b * 3 + c * 4 + d) / 10;
    printf ("Media: %.1lf\n", m);
    if (m >= 7) {
        printf ("Aluno aprovado.\n");
    }
    else if (m < 5) {
        printf ("Aluno reprovado.\n");
    }
    else if (m >= 5 && m <= 6.9) {
        printf ("Aluno em exame.\n");
        scanf ("%lf", &a);
        printf ("Nota do exame: %.1lf\n", a);
        m = (m + a) / 2;
        if (m >= 5) {
            printf ("Aluno aprovado.\n");
        }
        else {
            printf ("Aluno reprovado.\n");
        }
        printf ("Media final: %.1lf\n", m);
    }

    return 0;
}

File: 1-Iniciante/1040-Media-3/1040.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);
        float media = 0;
        for (int i = 0; i < 3; ++i) {
            media += scan.nextFloat()*(i+2);
        }
        media += scan.nextDouble();
        media /= 10;
        System.out.printf("Media: %.1f\n", media);
        if (media >= 5 && media < 6.9) {
            System.out.println("Aluno em exame.");
            float ex = scan.nextFloat();
            System.out.println("Nota do exame: " + ex);
            media = (media + ex) / 2;
            if (media >= 5) {
                System.out.println("Aluno aprovado.");
            }
            else {
                System.out.println("Aluno reprovado.");
            }
            System.out.printf("Media final: %.1f\n", media);
        }
        else if (media >= 7) {
            System.out.println("Aluno aprovado.");
        }
        else {
            System.out.println("Aluno reprovado.");
        }
    }
}